### DISCUSSION TITLE
Can CodeServer Get the storage in browser?

### DISCUSSION THREAD

**Discussion by slion-L** [coder/code-server#5361] [Q&A]
Can CodeServer Get the storage in browser?

**Comment by jsjoeio** [comment]
What are you trying to achieve? That will help us answer this.

**Reply by slion-L** [reply]
I want to code with serveral person in one code-server service and store something for each person. In web front end these stuff are usually stored in browser so i want to know if code-server can get the storage in browser by vscode extension.

**Reply by jsjoeio** [reply]
Ah I see. I believe if you built a web extension, you should be able to do this! Here are two resources:
- https://github.com/microsoft/vscode-extension-samples/tree/main/helloworld-web-sample
- https://code.visualstudio.com/api/extension-guides/web-extensions
- https://code.visualstudio.com/api/references/vscode-api#ExtensionContext.globalStorageUri

I don't see anything in the VS Code API for `localStorage`, but assuming you're in the browser, I would think you have access.
